projects/packages/connection/src/class-rest-connector.php

```diff
@@ -450,11 +450,14 @@ public function connection_register( $request ) {
 			}
 		}
 
-		return rest_ensure_response(
+		$response_body = apply_filters(
```

Comment:
Could you add a docblock here for the new filter?

Reply:
https://github.com/Automattic/jetpack/pull/19833/commits/e5b9dfc8d3417bdd72763ba33e91c78b24bcf270